File: 백준/Bronze/28235. 코드마스터 2023/코드마스터 2023.cc
#include <stdio.h>

int main(void) {
	char str[10];
	scanf("%s", str);
	switch (str[0]) {
	case 'S':
		printf("HIGHSCHOOL");
		break;
	case 'C':
		printf("MASTER");
		break;
	case '2':
		printf("0611");
		break;
	case 'A':
		printf("CONTEST");
		break;
	}
	return 0;
}

File: 백준/Bronze/28235. 코드마스터 2023/코드마스터 2023.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String str = br.readLine();
        switch(str.charAt(0)) {
            case 'S':
                System.out.println("HIGHSCHOOL");
                break;
            case 'C':
                System.out.println("MASTER");
                break;
            case '2':
                System.out.println("0611");
                break;
            case 'A':
                System.out.println("CONTEST");
                break;
        }
    }
}